lib/cuetip/models/job.rb

```diff
@@ -67,6 +67,8 @@ def execute(&block)
         # Initialize a new instance of the job we wish to execute
         job_klass = class_name.constantize.new(self)
 
+        Cuetip.config.emit(:started, self, job_klass)
+
```

Comment:
I think we should call this after the `expired` check and call it `before_perform`